BiweeklyMeetings/2022-08-16.md

```diff
@@ -0,0 +1,406 @@
+
+**Andrew** -: Yeah. We've had several two hour meetings with people. There was 20 different groups in the room, either producing music on Cardano or representing wallets or players or things like that. So yeah, I will add to the rationale section before next meeting that this comes up.
+
+**Matthias** -: Okay. Thanks.
+
+**Andrew** -: Probably not next meeting, but maybe the meeting after, because the next meeting is always in the times I can't be at.
+
+**Matthias** -: You're right. So let's keep that in for the US based next meeting.
+
+**Andrew** -: Yep, thank you.
+
+
+### Issues
+N/A
+
+### Close 
+
+**Matthias** -: So let's just wrap up quickly on what we just discussed. We've been through CIP 51 at the beginning, which we agreed to rediscuss, bring back on the table after [inaudible 00:48:13] happened. Since this anyway, won't happen before [inaudible 00:48:19] and further discussions are needed, then CIP 48 extended submitted data for, sorry, for CIP 25. CIP is a bit rushed out in terms of quality and format.
+So the content itself is okay technically-wise, but there are quite a few shortcomings when it comes to how it is written. So we ask the author to fix that. CIP 30, we have agreed to just close up requests. It's been inactive for a while. The author is not responsive, so I can't close it, unfortunately, because I'm on windows. But if any of the other auditors want to do it, please do. CIP 45 then the using pledge as a bidding program, there is some overlap with CIP 50. This is part of the broader discussions about keywords and incentives and there were concerns being raised about this actually incentivizing large pledge owners to just create [inaudible 00:49:30] without actually impacting, that is a tradition of Cardano. At least not in a good way.
+Then we went through CIP 68, which we agreed to put that last check for next week and merge as proposed given that's, the technical content was, already has been through many durations although there were a few points raised regarding this two-step beta data identifications that could perhaps, they discussed as well. So anyway, that will be a full two weeks before we go to the last check and we'll see for further updates after that. We've been through the NFT identity proposal to CIP 68, which also sounds technically okay, in the context of CIP 25, although it narrates the same shortcomings and flaws. So perhaps further discussions are needed. The format of that one is also a bit fleshed out and could deserve well, that be a bit more love and especially some more details on, well, the actual details of the CIPs. Possibly also some renaming, since this seems to be heavily inspired by the W3C recommendations, maybe could be made a bit clearer from the CIP title and finally CIP 60, the token metadata extension for music-related tokens, which comes as an extension of CIP 25 and has been a collaboration between many different individuals in the music industry around Cardano.
```

Comment:
```suggestion
Then we went through CIP 68, which we agreed to put that last check for next week and merge as proposed given that's, the technical content was, already has been through many iterations although there were a few points raised regarding this two-step beta data identifications that could perhaps, they discussed as well. So anyway, that will be a full two weeks before we go to the last check and we'll see for further updates after that. We've been through the NFT identity proposal to CIP 68, which also sounds technically okay, in the context of CIP 25, although it narrates the same shortcomings and flaws. So perhaps further discussions are needed. The format of that one is also a bit fleshed out and could deserve well, that be a bit more love and especially some more details on, well, the actual details of the CIPs. Possibly also some renaming, since this seems to be heavily inspired by the W3C recommendations, maybe could be made a bit clearer from the CIP title and finally CIP 60, the token metadata extension for music-related tokens, which comes as an extension of CIP 25 and has been a collaboration between many different individuals in the music industry around Cardano.
```